### PR TITLE
create apllication card

### PR DESCRIPTION
**Pull request by AhmedAbuSharar**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,3 +1,17 @@
-const App = () => <div className="App">hi</div>;
+import React from 'react';
+import ApplicationCard from './components/admin/ApplicationCard';
+
+const card = {
+  ownerName: 'ahmed',
+  ownerId: 1116,
+  licenseNumber: 123456,
+  pharmacyName: 'ahmed pharmacy',
+};
```
Comment (Raghad-Mezied) [inline client/src/App.tsx]: Move the type

File: client/src/components/admin/ApplicationCard.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+type Card = {
+  ownerName: string;
+  ownerId: number;
+  licenseNumber: number;
+  pharmacyName: string;
+};
+interface Props {
+  card: Card;
+}
```
Comment (Raghad-Mezied) [inline client/src/components/admin/ApplicationCard.tsx]: Update the imports and move the type

File: client/src/components/admin/ApplicationCard.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      <Typography sx={{ fontSize: '1.3rem', margin: '0.5rem 0' }}>
+        <strong>license number : </strong>
+        {card.licenseNumber}
+      </Typography>
+      <Typography sx={{ fontSize: '1.3rem', margin: '0.5rem 0' }}>
+        <strong>Pharmacy name : </strong>
+        {card.pharmacyName}
+      </Typography>
+    </Box>
+    <Box
```
Comment (Raghad-Mezied) [inline client/src/components/admin/ApplicationCard.tsx]: Why this card is here? isn't it in another pull request?

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Comment by AhmedAbuSharar** [comment]
image

**Review by Ibtisam-Hemmo — APPROVED** [review]

**Pull request merged**